zerver/lib/user_groups.py

```diff
@@ -938,7 +950,8 @@ def create_system_user_groups_for_realm(realm: Realm) -> dict[int, NamedUserGrou
         user_group = set_defaults_for_group_settings(group, {}, system_groups_name_dict)
         groups_with_updated_settings.append(user_group)
     NamedUserGroup.objects.bulk_update(
-        groups_with_updated_settings, ["can_add_members_group", "can_join_group", "can_manage_group", "can_mention_group"]
+        groups_with_updated_settings,
+        ["can_add_members_group", "can_join_group", "can_manage_group", "can_mention_group"],
```

Comment:
I assume this line-wrapping is a linter fix that belongs in the previous commit.

Reply:
Fixed!